Project.toml

```
@@ -1,7 +1,7 @@
 name = "OpenFHE"
 uuid = "77ce9b8e-ecf5-45d1-bd8a-d31f384f2f95"
 authors = ["Michael Schlottke-Lakemper <michael@sloede.com>"]
-version = "0.1.9-dev"
+version = "0.2.0"
```

```suggestion
version = "0.1.9"
```
What is the breaking change here (honest question, maybe I missed it)?

Note that I wouldn't consider the update of the dependencies a breaking change

---

I thought exactly as with `openfhe_julia`, we updated dependencies on `libcxxwrap_julia` and `CxxWrap.jl` that are not compatible with oldest version, may be I misunderstood definition of breaking changes...

---

Yes, the water is a little muddy here. However, also in Trixi.jl we tend not to make a breaking change for a new release only because of breaking releases in dependencies. With openfhe-julia it was motivated because it caused us so much grief to get it right, that I wanted to make sure nobody _accidentally_ depended on the newer version.